src/html.js

```diff
@@ -0,0 +1,36 @@
+import React from "react";
```

Comment:
What/Why exactly do we need this file for?

Reply:
This file was created for server-side rending of a dark theme, Yes, but here It is not necessary, We can remove this for now.

Reply:
as we are using a mounting, we don't need this.